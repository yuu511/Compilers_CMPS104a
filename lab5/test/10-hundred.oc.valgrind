==24192== Memcheck, a memory error detector
==24192== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==24192== Using Valgrind-3.14.0 and LibVEX; rerun with -h for copyright info
==24192== Command: ../oc 10-hundred.oc
==24192== 
==24192== 
==24192== HEAP SUMMARY:
==24192==     in use at exit: 0 bytes in 0 blocks
==24192==   total heap usage: 397 allocs, 397 frees, 140,215 bytes allocated
==24192== 
==24192== All heap blocks were freed -- no leaks are possible
==24192== 
==24192== For counts of detected and suppressed errors, rerun with: -v
==24192== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
