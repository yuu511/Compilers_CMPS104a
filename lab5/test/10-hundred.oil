main:      .function int 
           .local int count
           count = 0  
.wh0:      goto .od0 if count > 16
.do0:      $t0:i = count + 1
           count = $t0:i  
           call putint(count)  
           call putchr('\n')  
           goto .wh0  
.od0
           return 0
           return
           .end
