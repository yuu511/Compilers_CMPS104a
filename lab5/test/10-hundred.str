string_set[   1]:   11597697714117577063 0x55fc21cb6838->"="
string_set[   2]:    5344662657899890615 0x55fc21cb49b8->";"
string_set[   3]:   10159970873491820195 0x55fc21cb7078->"1"
                    1396094175042499165 0x55fc21cb6b98->"<"
                   10319632053171904911 0x55fc21cb6a58->"while"
string_set[   4]:    9201800968376702500 0x55fc21cb5b78->"getchr"
string_set[   7]:   13348923631108115385 0x55fc21cb5698->"putstr"
string_set[   8]:    9611428142580219387 0x55fc21cb40b8->"fail"
string_set[   9]:   18413295556186398326 0x55fc21cb5038->"putint"
string_set[  10]:    6192799629375248351 0x55fc21cb7438->"'\n'"
                   16579916199511856337 0x55fc21cb6178->"getln"
                   15816506932648033945 0x55fc21cb41b8->"("
string_set[  11]:   13686721134187162920 0x55fc21cb7548->"}"
string_set[  12]:   11868120863286193964 0x55fc21cb6478->"main"
                   12679825168387489523 0x55fc21cb51f8->"num"
string_set[  13]:   11166701219533395393 0x55fc21cb5e78->"getstr"
                    4706206694200543916 0x55fc21cb40f8->","
string_set[  14]:    1583576014053361350 0x55fc21cb5858->"str"
                    7296690267450279874 0x55fc21cb4cd8->"chr"
                    4017852018407011111 0x55fc21cb45d8->"file"
string_set[  15]:    7282951971071689374 0x55fc21cb6c58->"16"
                   12291392672316406109 0x55fc21cb4818->"line"
string_set[  16]:   12868445110721718657 0x55fc21cb48f8->")"
string_set[  20]:    9386170080689500517 0x55fc21cb4338->"expr"
string_set[  21]:    2297668033614959926 0x55fc21cb68f8->"0"
string_set[  22]:    1782657156151550167 0x55fc21cb6778->"count"
string_set[  26]:   14635384904416252381 0x55fc21cb7588->"return"
string_set[  27]:    1370730858159036685 0x55fc21cb6fb8->"+"
                   12597418120160561823 0x55fc21cb4758->"int"
string_set[  28]:    7211130878751977441 0x55fc21cb4b18->"putchr"
string_set[  29]:    3729804957429652673 0x55fc21cadcc8->"<<ROOT>>"
string_set[  32]:      21606623943142546 0x55fc21cb6638->"{"
string_set[  33]:    6142509188972423790 0x55fc21cb43f8->""
string_set[  35]:   11911796782241914524 0x55fc21cb2fe8->"void"
string_set[  36]:   13497578857207247412 0x55fc21cb4278->"string"
load_factor = 0.946
bucket_count = 37
max_bucket_size = 3
