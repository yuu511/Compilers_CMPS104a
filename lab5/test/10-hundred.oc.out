CONDITIONAL WHILE STATEMENT SYMBOL TOK_LT 
